### PR TITLE
Shawnjj rebase

### PR DESCRIPTION
**Pull request by sjacks49**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/CronServlet.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        // Filters out Posted Deeds
+        List<GoodDeed> GoodDeeds_cleaned = cleanDeeds(GoodDeeds);
+
+        if (GoodDeeds_cleaned.size() < MINIMUM_QUERY_LENGTH) {
+            System.out.println("Query is below minimum size.");
+            
+            // Resets Posted Yet property of all posted deeds
+            resetPostedYet();
+            GoodDeeds_cleaned = GoodDeeds;
+            //System.out.println(GoodDeeds_cleaned);
```
Comment (ShreeragJ) [inline src/main/java/com/google/sps/servlets/CronServlet.java:67]: Remove this line

File: src/main/webapp/WEB-INF/cron.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cron:
+    - description: "Choose Daily Deed"
+    - url: /cronjob
+    - schedule: every 2 minutes
```
Comment (ShreeragJ) [inline src/main/webapp/WEB-INF/cron.yaml:4]: Is this the right interval?

File: src/main/java/com/google/sps/servlets/CronServlet.java (hunk 1 above)
Comment (sjacks49) [reply src/main/java/com/google/sps/servlets/CronServlet.java:67]: @ShreeragJ the System.out.print line?

Comment (ShreeragJ) [reply src/main/java/com/google/sps/servlets/CronServlet.java:67]: Yes

Comment (sjacks49) [reply src/main/java/com/google/sps/servlets/CronServlet.java]: @ShreeragJ Got it. I removed all of the debugging print statements

**Pull request merged**